rockcraft/__init__.py

```diff
@@ -16,9 +16,12 @@
 
 """The craft tool to create ROCKs."""
 
-from craft_parts import Features
```

Comment:
The reason that I moved this from here to the lifecycle service is because we do `python3 setup.py --version` without installed deps to get the version at snap-build-time, so this import fails.
I think it makes more sense to be in the lifecycle service anyway